src/helpers/iconpicker.class.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+
+var $           = require('jquery');
+var eventablejs = require('eventablejs');
+var _           = require('../lodash.js');
+var Modal       = require('etudiant-mod-modal');
+var xhr         = require('etudiant-mod-xhr');
+var Slider      = require('./slider.class.js');
+var FileUploader = require('../extensions/file-uploader.js');
+
+var iconPickerHtml = '<div class="icon-picker"><div class="droppable st-block__upload-container">' + i18n.t('blocks:illustrated:placeholder:drop') + '</div></div>';
```

Comment:
@arthur-balagne don't hesitate to use methods like `Array.join('\n');` to make your templates more readable